python_modules/libraries/dagster-dg/dagster_dg/cli/init.py

```diff
@@ -46,12 +46,28 @@ def init_command(
     project_python_environment: DgProjectPythonEnvironment,
     **global_options: object,
 ):
-    """Initialize a new Dagster workspace and a first project within that workspace.
+    """Initialize a new Dagster project or workspace.
 
-    The scaffolded workspace folder has the following structure:
+    By default, only a standalone project is created. A workspace can also be created using
+    the --workspace-name option.
+
+    If a project is created, it will have the following structure:
+
+    \b
+    ├── <project name>
+    │   ├── __init__.py
```

Comment:
This needs to be updated with new layout

Reply:
this matches what we currently have in the docs which is programtically generated by scaffolding a project and then running 'tree' - is the new layout that you're referring to live in master yet? https://github.com/dagster-io/dagster/blob/15e761fbf6f774832cfea954ebce774a47e7b15e/examples/docs_snippets/docs_snippets/guides/dg/scaffolding-project/2-tree.txt#L1-L18

Reply:
errr wait ignore me, no it doesn't!